### PR TITLE
Enhanced Music Portfolio: Added Image Animation and Appearance Tweaks

### PR DESCRIPTION
**Pull request by tmrsich**
Changes made:
- Deleted about-me.css because those styles were identical to those in index.css
	- Changed a <link> tag to reference index.css instead of about-me.css
- Added a border-radius property to the audio containers
- Separated "8-bit CD.png" and "Image.jpeg" on the homepage to be in separate containers
	- Moved the CD image to be in the upper left corner of the page
	- Created an animation for the CD Image
- Added margin/padding to various elements for appearance
        - Gave cd-image-container a top of 0
- Changes to media queries:
	- On devices with a viewport width > 768px:
		- Set the margin property for the cd-image-container class
		- Inserted back into the regular document flow
		- Set a max width for the cd image
		- Gave it a margin-top of 25px so the cd image isn't obstructed by the main title
	- On devices with a viewport width < 768px:
		- Set the cd-image height to 75px
		- Gave it a position of relative so it will be centered above the image (inserted back into the regular document flow)
		- Centered the image by giving its _container_ a display of flex followed by a justify-content: center
		- 10px from the top
TL;DR:
- Deleted about-me.css
- Appearance changes (border-radius, image animations)
- Margin/padding adjustments
- Changes to media queries

### CODE REVIEW

**Review by timrolsh — APPROVED** [review]
Appreciate the changes and the re-organization of the codebase, I really like the animation that was added and how you made it responsive automatically. Straight forward changes and good documentation added, LGTM and merging
